R/plumber.R

```diff
@@ -182,12 +182,14 @@ plumber <- R6Class(
 
     },
     run = function(host='127.0.0.1', port=getOption('plumber.port'), swagger=interactive(),
-                   debug=interactive()){
+                   debug= getOption('plumber.debug') %||% interactive() ){
```

Comment:
Oh, I see it's custom. I feel like I'm going to get tripped up on this every time I come across it and I think there's a bug in the implementation. And since this is part of the public function signature I'd rather not have this internal function leak out.

Can we just ignore the option here and leave it as `interactive()`? Then if someone wants to override it they can do so in the `run()` invocation.